readthedocs/templates/search/base_facet.html

```diff
@@ -24,7 +24,7 @@
           <input type="hidden" name="selected_facets" value="{{ selected_facets }}" id='id_selected_facets'>
         </div>
         <div class="submit-input-wrapper">
-          <input type="submit" value="{% trans "Go crazy." %}" id="id_search_button">
+          <input type="submit" value='{% trans "Search" %}' id="id_search_button">
```

Comment:
You shouldn't change the double quotes to single quotes here, html needs `"`.

Reply:
<code>"{% trans 'search' %}" </code> 
This syntax will do i guess. I think both shall not be double quotes.. Do correct me if i'm wrong